packages/fetchai/skills/erc1155_deploy/generic_data_model.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2020 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""This package contains the dataModel for the erc1155 deploy skill aea."""
+from typing import Any, Dict
+
+from aea.helpers.search.models import Attribute, DataModel
+
+SUPPORTED_TYPES = {"str": str, "int": int, "float": float, "bool": bool}
+
+
+class GenericDataModel(DataModel):
```

Comment:
We should extract this into `aea.helpers.search.models` and then use it from there rather than have it here. It is generic after all.